<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ElecappliancesPanel</class>
 <widget class="QDialog" name="ElecappliancesPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>301</width>
     <height>471</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelImg">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>221</width>
     <height>191</height>
    </rect>
   </property>
   <property name="text">
    <string>ElecapplianceImage</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelName">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>220</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>ElecapplianceName</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelIntro">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>250</y>
     <width>221</width>
     <height>181</height>
    </rect>
   </property>
   <property name="text">
    <string>Intro</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>460</y>
     <width>221</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
